src/app/modules/main/wallet_section/collectibles/models/collectibles_model.nim

```diff
@@ -31,6 +38,8 @@ QtObject:
     new(result, delete)
     result.setup
     result.items = items
+    result.allCollectiblesLoaded = false
+    result.isFetching = true
```

Comment:
These should use the setters, to get the property bindings on QML working (you miss the notifier signal this way)

Reply:
hmm I thought signaling wasn't necessary for the initialization. My understanding was that the QML binding assigned the initial value through the getter and then connected to the signal, waiting for changes.